src/index.html

```diff
@@ -5,8 +5,322 @@
+                </div>
+              </li>
+
+              <li class="project-card">
+                <img src="./assets/images/oregano-height.png" alt="Oregano Height picture" class="projects-card__img">
+                <div class="projects-card__descr">
+                  <span class="projects-card__descr-title">Oregano Height</span>
+                  <span class="projects-card__descr-address">2464 Royal Ln. Mesa, New Jersey</span>
+                </div>
+              </li>
+
+            </ul>
+            <div class="tabs__navigation">
+              <button class="btn btn-lg btn-next btn-dark">Next
+                <svg class="btn__icon icon--next" fill="#fff">
+                  <use xlink:href="#arrow-next"/>
+                </svg>
+              </button>
+              <div class="dots">
+                <div class="dots__item"></div>
+                <div class="dots__item dotes__item--active"></div>
+                <div class="dots__item"></div>
+                <div class="dots__item"></div>
+                <div class="dots__item"></div>
+              </div>
+              <button class="btn btn-lg btn-prev btn--dark">Back
+                <svg class="btn__icon icon--prev" fill="#fff">
+                  <use xlink:href="#arrow-back"/>
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
+      </section>
+<!-- ------------------ FEEDBACK --------------------- -->
+      <section class="feedback">
+        <h2 class="feedback__title">What can us do for you?</h2>
+        <p class="feedback__description">We are ready to work on a project of any complexity, whether it&#39;s commercial or residential.</p>
+        <form action="#" method="post" class="feedback__form">
+          <div class="feedback__form-fields">
+            <input type="text" placeholder="Your name" class="field" required />
+            <input type="email" placeholder="Email" class="field" required />
+            <input type="text" placeholder="Reason for Contacting" class="field" required />
+            <input type="tel" placeholder="Phone" class="field" />
+          </div>
+          <textarea cols="30" rows="10" placeholder="Message" class="textarea"></textarea>
+          <span class="feedback__form-note">* indicates a required field</span>
+          <button type="submit" class="btn btn--blue btn-sm" href="#">Submit</button>
+        </form>
+      </section>
+<!-- ------------------ CONTACTS --------------------- -->
+      <section class="contacts">
+        <div class="contacts__info">
+          <span class="contacts__info-item"><span class="contacts__info-name">ADDRESS:</span>6391 Elgin St. Celina, Delaware 10299</span>
+          <span class="contacts__info-item"><span class="contacts__info-name">PHONE:</span>6391 Elgin St. Celina, Delaware 10299</span>
+          <span class="contacts__info-item"><span class="contacts__info-name">EMAIL:</span>hello@thebox.com</span>
```

Comment:
`hello@thebox.com` is a link `<a href="mailto:YUR_MAIL">`

Reply:
Fixed